pandas/core/groupby.py

```diff
@@ -37,27 +38,18 @@
 import pandas.algos as _algos
 import pandas.hashtable as _hash
 
-_agg_doc = """Aggregate using input function or dict of {column -> function}
-
-Parameters
-----------
-arg : function or dict
-    Function to use for aggregating groups. If a function, must either
-    work when passed a DataFrame or when passed to DataFrame.apply. If
-    passed a dict, the keys must be DataFrame column names.
-
-Notes
------
-Numpy functions mean/median/prod/sum/std/var are special cased so the
-default behavior is applying the function along axis=0
-(e.g., np.mean(arr_2d, axis=0)) as opposed to
-mimicking the default Numpy behavior (e.g., np.mean(arr_2d)).
-
-Returns
--------
-aggregated : DataFrame
-"""
+_doc_template = """
+
+        Returns
+        -------
+        same type as input
```

Comment:
I find this a bit unclear what this means in the case of a groupby. I would personally leave it out.

Eg for `Groupby.mean`, how to interpret this? There is not really an 'input', apart from the object on which you call this. But that is a GroupBy object, and that is not what you get back. 
For groupby is also possible that you start with a DataFrame, but not get a DataFrame as the result.

(for `rolling` this is indeed more appropriate)